### ISSUE TITLE
Bind with child domain creds and search sibling domains not working

### ISSUE THREAD

**Issue by nspaul** [Adldap2/Adldap2#580]
- Adldap2 Version: 9.0.1
- LDAP Type: Active Directory
- PHP Version: 7.0.30

### Description:
I am working towards updating Adldap from version 7.0.15 to 9.0.1. I have made the appropriate changes to the configuration, and I am connecting to AD just fine. However, my search scope seems to be limited to the bind account I am using. It only finds users in the same domain as the bind account. This was not the case in adldap version 7.

### Steps To Reproduce:
Set up a super basic page that includes Adldap version 9 and binds with an account in a child domain. Perform a search similar to this one:
```
$user = $provider->search()->findBy('samaccountname',$userid);
echo $user->getDisplayName()
```
It will only return a user for an account in the same domain as the bind account. Switch your search query to a user in another domain, and it will fail. Then, change your bind account to an account in the same domain as the users in the new search query, and you will find a successful query again.

Now, do the same steps using Adldap version 7, and it will succeed on searches for users in domains that aren't necessarily the same domain as the bind account.

**Comment by stevebauman** [comment]
Can you also try the following to see if you receive results?

```php
$usersInAlpha = $provider->search()->in('dc=alpha,dc=com')->users()->get();

$usersInBeta = $provider->search()->in('dc=beta,dc=com')->users()->get();
```

**Comment by nspaul** [comment]
I'm glad we had this discussion, because it forced me to really simplify things down to the essence, and I must have been leaving out a detail somewhere. As I was preparing my response to this, I got the code to function as expected.

To address the questions posed, though... the base_dn didn't change at all. I was just moving from version 7 to version 9, and I had made the required changes (such as changing admin_username to username). Also, my use case is to throw a username at a forest (say, company.org), and it should automatically find the user regardless of what child domain (alpha.company.org or beta.company.org) the user is in.

Once I got this working as expected, I then moved on to "upgrading" from port 3268 to 3269, so that we are talking over ssl. This ultimately worked out just fine for me.

So all of this thread is rather moot at this point. But I wanted to make these statements here just to close this issue out and have a history recorded.

Thank you for maintaining this package! It's amazing.
